docs/cryptodoc/src/05_09_kyber.rst

```diff
@@ -1,138 +1,299 @@
 .. _pubkey/kyber:
```

Comment:
```suggestion
.. _pubkey/ml_dsa:
```

... no strong feeling, but would be more future-oriented.

Reply:
For sure!